### PR TITLE
GRAPHICS: Support flipping in scaleBlit and rotoscaleBlit

### PR DESCRIPTION
**Pull request by ccawley2011**

### CODE REVIEW

File: graphics/transform_struct.h [hunk 1]
```diff
@@ -35,6 +35,22 @@ enum TSpriteBlendMode {
...
+	/// The image will not be flipped.
+	FLIP_NONE = 0,
+	/// The image will be flipped at the horizontal axis.
+	FLIP_H = 1,
+	/// The image will be flipped at the vertical axis.
+	FLIP_V = 2,
+	/// The image will be flipped at the horizontal and vertical axis.
+	FLIP_HV = FLIP_H | FLIP_V,
+	/// The image will be flipped at the horizontal and vertical axis.
+	FLIP_VH = FLIP_H | FLIP_V
```
Comment (bluegr) [inline graphics/transform_struct.h:51]: Isn't that the same as FLIP_HV above?

Comment (ccawley2011) [reply graphics/transform_struct.h:51]: It is, but it already existed before when the enum was in `graphics/transparent_surface.h`. Should I remove it anyway?

Comment (sev-) [reply graphics/transform_struct.h:51]: It is just a convenience thing. Let it stay

**Comment by sev-** [comment]
Thank you!

**Pull request merged**